### ISSUE TITLE
Support multiple languages

### ISSUE THREAD

**Issue by munanadi** [GokiProtocol/walletkit#91]
Are there plans to include multiple languages?
Would you accept  a PR if I used https://react.i18next.com/ ? 

Is there a nice way to do it? Lazy load translations? Anything I can start working with? 

PS. Also is there a local dev workflow to make changes in `walletkit` and use the example dir to see these changes. 
Sorry noob here.

**Comment by Aadhinana** [comment]
This is no where close to a PR
But here it goes. https://github.com/Aadhinana/walletkit

I really like to to contribute to this. Please help.

**Comment by sungoki** [comment]
Thanks @Aadhinana. I'll see how we can include the language selection in the UI-- we should see if we can make it automatically detect the language like Uniswap does.

**Comment by Aadhinana** [comment]
Yes if the `langOptions` are left empty then by default it shows only english as the sole option.
like this 
![wihtoutLangOptions](https://user-images.githubusercontent.com/22261173/138132228-0f429dd2-13e2-4fc4-927a-22c27fb7ba19.gif)

I thought I'll ask you about the styling. Since it should be consistent with the whole thing.
Okay sure! :+1: 
Thanks!

PS: I would like to contribute more! Let me know what's next!
